.github/workflows/monitor_agricultural_hotspots.yaml

```diff
@@ -0,0 +1,62 @@
+name: Monitor JRC agricultural hotspots
+
+on:
+    workflow_dispatch:
+      inputs:
+        first_run:
+          required: true
+          type: choice
+          default: "FALSE"
+          options:
+            - "TRUE"
+            - "FALSE"
+        test:
+          required: true
+          type: choice
+          default: "TRUE"
+          options:
+            - "TRUE"
+            - "FALSE"
+        GMAS_TEST_RUN:
+          required: true
+          type: choice
+          default: "TRUE"
+          options:
+            - "TRUE"
+            - "FALSE"
+        LOG_LEVEL:
```

Comment:
Hah, good question. I didn't really understand `TRACE`, tbh, so just used what I was used to.

Reply:
I'd be interested if it does work, cause tracing is the one thing I'm quite familiar with because when you encounter an error in the R console, generating the traceback of function calls is an easy way to trace where something went wrong. However, `{box}` is why some of the error throws have limited history, so probably wouldn't be that useful in this instance.

Reply:
Ahh interesting 🤔 I'll take a look into it